src/common/transformations/src/transformations/op_conversions/convert_slicescatter.cpp

```diff
@@ -0,0 +1,76 @@
+        if (!slice_node) {
+            return false;
+        }
+        NodeRegistry node_registry;
+        auto const_0 = node_registry.make<ov::op::v0::Constant>(ov::element::i64, Shape{}, 0);
+        auto const_1 = node_registry.make<ov::op::v0::Constant>(ov::element::i64, Shape{}, 1);
+        auto const_1d_neg_1 =
+            node_registry.make<ov::op::v0::Constant>(ov::element::i64, Shape{1}, std::vector<int64_t>{-1});
+        auto const_scatter_indices_shape =
+            node_registry.make<ov::op::v0::Constant>(ov::element::i64, Shape{2}, std::vector<int64_t>{-1, 1});
+        auto data_shape = node_registry.make<ov::op::v3::ShapeOf>(slice_node->input_value(0), ov::element::i64);
+        auto num_elements_data = node_registry.make<ov::op::v1::ReduceProd>(data_shape, const_0, false);
+        auto data_indices_flatten =
+            node_registry.make<ov::op::v4::Range>(const_0, num_elements_data, const_1, ov::element::i64);
+        auto full_data_indices = node_registry.make<ov::op::v1::Reshape>(data_indices_flatten, data_shape, false);
+        std::shared_ptr<ov::op::v8::Slice> slice_indices;
+        if (slice_node->get_input_size() == 5) {
+            slice_indices = node_registry.make<ov::op::v8::Slice>(full_data_indices,
+                                                                  slice_node->input_value(2),
+                                                                  slice_node->input_value(3),
+                                                                  slice_node->input_value(4));
+        } else {
+            slice_indices = node_registry.make<ov::op::v8::Slice>(full_data_indices,
+                                                                  slice_node->input_value(2),
+                                                                  slice_node->input_value(3),
+                                                                  slice_node->input_value(4),
+                                                                  slice_node->input_value(5));
+        }
+        auto slice_indices_flatten =
+            node_registry.make<ov::op::v1::Reshape>(slice_indices, const_scatter_indices_shape, false);
+        auto updates_flatten =
+            node_registry.make<ov::op::v1::Reshape>(slice_node->input_value(1), const_1d_neg_1, false);
+        auto data_flatten = node_registry.make<ov::op::v1::Reshape>(slice_node->input_value(0), const_1d_neg_1, false);
+        auto output_flatten =
+            node_registry.make<ov::op::v3::ScatterNDUpdate>(data_flatten, slice_indices_flatten, updates_flatten);
+        auto output = node_registry.make<ov::op::v1::Reshape>(output_flatten, data_shape, false);
+
+        output->set_friendly_name(slice_node->get_friendly_name());
+        copy_runtime_info(slice_node, node_registry.get());
+        replace_node(slice_node, output);
+        slice_node->clear_control_dependencies();
```

Comment:
why do we need to clear control dependencies here?

Reply:
Removed, forgot to delete after checking this function